packages/aws-cdk-lib/aws-codecommit/README.md

```diff
@@ -27,6 +27,21 @@ declare const repo: codecommit.Repository;
 repo.notify('arn:aws:sns:*:123456789012:my_topic');
 ```
 
+## Use customer managed key
+
+CodeCommit repositories are automatically encrypted with an AWS managed key. You can also [use a customer managed key](https://docs.aws.amazon.com/codecommit/latest/userguide/encryption.html) for encryption by specifying the `kmsKey` property.
```

Comment:
I tried to make the link name match the title of the page destination and added a line break.

```suggestion
CodeCommit repositories are automatically encrypted with an AWS managed key. You can also use
a customer managed key for encryption by specifying the `kmsKey` property,

For more information, see
[AWS Key Management Service and encryption for AWS CodeCommit repositories](https://docs.aws.amazon.com/cdk/latest/guide/reference.html#versioning).
```